regression-tests.recursor-dnssec/test_Chain.py

```diff
@@ -0,0 +1,30 @@
+import dns
+import os
+import time
+from recursortests import RecursorTest
+
+class ChainTest(RecursorTest):
+    _confdir = 'Chain'
+
+    _config_template = """dnssec=validate
+"""
+
+    def testBasic(self):
+        count = 500
```

Comment:
May I suggest adding the commit title (`hits the case of #14624`) as a comment here?